src/components/AuthenticatedApp/AuthenticatedApp.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { renderWithRouter } from 'testUtils';
+import { userData, authenticationHeaders } from 'testUtils/mocks/auth';
+import '@testing-library/jest-dom/extend-expect';
+
+import AuthenticatedApp from './AuthenticatedApp';
+
+const state = {
+  auth: {
+    user: userData,
+    session: authenticationHeaders,
+  },
+};
+
+describe('Authenticated App', () => {
+  it('renders the Home Page', () => {
+    const { getByText } = renderWithRouter(<AuthenticatedApp />, {
+      state,
+      history: ['/'],
+    });
+    expect(getByText('Hello World')).toBeInTheDocument();
+  });
+  it('renders the Settings Page', () => {
+    const { getByText } = renderWithRouter(<AuthenticatedApp />, {
+      state,
+      history: ['/settings'],
+    });
+    expect(getByText('Account settings')).toBeInTheDocument();
+  });
+  it('renders the No match Page', () => {
+    const { getByText } = renderWithRouter(<AuthenticatedApp />, {
+      state,
+      history: ['/no-match'],
+    });
+    expect(getByText('404')).toBeInTheDocument();
+  });
```

Comment:
maybe check that some unauthenticated page (like sign-in) redirects to the authenticated app?

Reply:
sorry, this isn't accurate... that test should be in App.test.js